PirateGame/Assets/Scripts/UserShipAgent.cs

```diff
@@ -21,13 +20,10 @@ void Start ()
 			done in update: open shop menu for user to purchase item, game timer starts when menu is exited
 		 */
 		
-		ship = (Ship) this.GetComponent(typeof(Ship));
-		//gui = GameObject.Find ("PlayerShip").GetComponent <GUIFunctions>();
-
+		ship = GetComponent<Ship>();
+		gui = ((GameObject)GameObject.Find("GUI_Manager")).GetComponent<GUIFunctions>();
 		screenText = "";
-		ship.state = Ship.State.Roaming;
-		//init ship's harbor
-		harbor = GameObject.Find ("PlayerHarbor").transform;
+		ship.state = Ship.State.Roaming; //Should probably start out as shopping & facing the harbor so they can get their first free upgrade when the game starts for the first time.
```

Comment:
comments like this should be sent in an email or left as a TODO comment, otherwise noone may see it